test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -2,82 +2,117 @@
-		set-alias abcd01 efgh01
-		set-alias abcd02 efgh02
-		set-alias abcd03 efgh03
-		set-alias abcd04 efgh04
-		set-alias ijkl01 mnop01
-		set-alias ijkl02 mnop02
-		set-alias ijkl03 mnop03
-		set-alias ijkl04 mnop04
+        New-Item -Path $testAliasDirectory -ItemType Directory -Force
+        remove-item alias:abcd* -force -ErrorAction SilentlyContinue
+        remove-item alias:ijkl* -force -ErrorAction SilentlyContinue
+        set-alias abcd01 efgh01
+        set-alias abcd02 efgh02
+        set-alias abcd03 efgh03
+        set-alias abcd04 efgh04
+        set-alias ijkl01 mnop01
+        set-alias ijkl02 mnop02
+        set-alias ijkl03 mnop03
+        set-alias ijkl04 mnop04
     }
 
-	AfterEach {
-		Remove-Item -Path $testAliasDirectory -Recurse -Force -ErrorAction SilentlyContinue
-	}
+    AfterEach {
+        Remove-Item -Path $testAliasDirectory -Recurse -Force -ErrorAction SilentlyContinue
+    }
 
-	It "Import-Alias Resolve To Multiple will throw PSInvalidOperationException" {
-		{ Import-Alias * -ErrorAction Stop } | Should -Throw -ErrorId "NotSupported,Microsoft.PowerShell.Commands.ImportAliasCommand"
-	}
+    It "Import-Alias Resolve To Multiple will throw PSInvalidOperationException" {
+        { Import-Alias * -ErrorAction Stop } | Should -Throw -ErrorId "NotSupported,Microsoft.PowerShell.Commands.ImportAliasCommand"
+    }
 
-	It "Import-Alias From Exported Alias File Aliases Already Exist should throw SessionStateException" {
-		{ Export-Alias  $fulltestpath abcd* } | Should -Not -Throw
-		{ Import-Alias $fulltestpath -ErrorAction Stop } | Should -Throw -ErrorId "AliasAlreadyExists,Microsoft.PowerShell.Commands.ImportAliasCommand"
+    It "Import-Alias From Exported Alias File Aliases Already Exist should throw SessionStateException" {
+        { Export-Alias  $fulltestpath abcd* } | Should -Not -Throw
+        { Import-Alias $fulltestpath -ErrorAction Stop } | Should -Throw -ErrorId "AliasAlreadyExists,Microsoft.PowerShell.Commands.ImportAliasCommand"
     }
 
-	It "Import-Alias Into Invalid Scope should throw PSArgumentException"{
-		{ Export-Alias  $fulltestpath abcd* } | Should -Not -Throw
-		{ Import-Alias $fulltestpath -scope bogus } | Should -Throw -ErrorId "Argument,Microsoft.PowerShell.Commands.ImportAliasCommand"
+    It "Import-Alias Into Invalid Scope should throw PSArgumentException"{
+        { Export-Alias  $fulltestpath abcd* } | Should -Not -Throw
+        { Import-Alias $fulltestpath -scope bogus } | Should -Throw -ErrorId "Argument,Microsoft.PowerShell.Commands.ImportAliasCommand"
     }
 
-	It "Import-Alias From Exported Alias File Aliases Already Exist using force should not throw"{
-		{Export-Alias  $fulltestpath abcd*} | Should -Not -Throw
-		{Import-Alias $fulltestpath  -Force} | Should -Not -Throw
+    It "Import-Alias From Exported Alias File Aliases Already Exist using force should not throw"{
+        {Export-Alias  $fulltestpath abcd*} | Should -Not -Throw
```

Comment:
```suggestion
        { Export-Alias  $fulltestpath abcd* } | Should -Not -Throw
```

Reply:
this file shouldn't have changed at all in this PR, I'll try to revert it back